quiz/web/.eslintrc.json

```diff
@@ -32,6 +32,7 @@
                 "extensions": [".tsx", ".ts"]
             }
         ],
+        "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
```

Comment:
Will stop eslint from throwing errors about packages not present in dependencies beacause they are in devDeps.